sdks/python/agenta/sdk/agents/handler.py

```
@@ -76,21 +75,16 @@ def _default_template() -> AgentTemplate:
 
-    `local` (unconfined host bash, not a tenant boundary) is refused unless
-    `AGENTA_SANDBOX_LOCAL_ALLOWED` is on — a bare `agent_v0` gets this protocol-level
-    safety behavior for free, same as capability gating and MCP gating above.
+    A sandbox the deployment has not enabled is refused before any run — a bare `agent_v0`
+    gets this protocol-level safety behavior for free, same as capability gating and MCP
+    gating above. The runner remains the final authority; this is a pre-filter that reads the
+    same `AGENTA_RUNNER_ENABLED_SANDBOX_PROVIDERS` registry.
     """
-    if agent_template.sandbox == "local" and not _sandbox_local_allowed():
-        raise LocalSandboxNotAllowedError()
+    if not sandbox_provider_enabled(agent_template.sandbox):
```

Flake8 (7.3.0)

[error] undefined name 'sandbox_provider_enabled'

(F821)